integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -236,9 +254,9 @@ def test_multiple_granules_happy_path(self):
                 "Expected API output not returned.",
             )
             # Make sure all given granules are present without checking order.
-            self.assertEqual(
-                len(expected_catalog_output_granules),
-                len(catalog_output_json["granules"]),
+            self.assertCountEqual(
```

Comment:
Turns out this does more than I remembered. It checks the content as well as count. The error messages are just awful.